.codeclimate.yml

```diff
@@ -1,4 +1,11 @@
 version: "2"
+plugins:
+  duplication:
+    enabled: true
+    config:
+      languages:
+        filters:
+          - "(object (Identifier PropTypes))"
```

Comment:
Based on https://github.com/codeclimate/codeclimate-duplication/issues/89

Reply:
Would be really nice if this works, but I still see 6 fixed issues on codeclimate, there should only be the two propTypes that are fixed

Reply:
okay, look into it

Reply:
now it is correct

Reply:
I don't know why, but if you look at codeclimate it is saying that the propTypes duplications are fixed, then there are two new issues that are exactly the same... We can try to merge this and see what it looks like when ir runs on main

Reply:
sure